config/environments/development.rb

```diff
@@ -11,11 +11,14 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = true
+  config.eager_load = false
```

Comment:
I didn't see any reason why this (still?) needed to be on for development, so I set this back to false to speed up booting.